app/views/unattended/provisioning_templates/snippet/ansible_tower_callback_script.erb

```diff
@@ -10,9 +10,17 @@ description: |
 #!/bin/sh
 
 echo "Calling Ansible AWX/Tower provisioning callback..."
+
+# Query the AC version
+if [ $(echo "$(curl -k -s https://<%= host_param('ansible_tower_fqdn') %>/api/v2/ping/ | jq -r .version | cut -d. -f1,2) >= 4.6" | bc) -eq 1 ]; then
```

Comment:
Do I need to use `/usr/bin/curl` here, as in the lines below?

Reply:
It makes sense to follow the same pattern i.e. use of the full binary path. So +1 ..